modules/acquia_cms_site_studio/config/optional/editor.editor.cohesion.yml

```diff
@@ -359,7 +390,7 @@ image_upload:
   status: true
   scheme: public
   directory: inline-images
-  max_size: ''
+  max_size: '5 MB'
```

Comment:
The default max size for image upload in Drupal is 2MB, Any reason we are making it 5MB?
cc @rajeshreeputra @vishalkhode1

Reply:
Updated.